Client/qt/dialog.ui

```diff
@@ -0,0 +1,241 @@
+        </property>
+      </widget>
+      <widget class="QCheckBox" name="focusOnVoiceCheckBox">
+        <property name="enabled">
+          <bool>false</bool>
+        </property>
+        <property name="geometry">
+          <rect>
+            <x>10</x>
+            <y>110</y>
+            <width>111</width>
+            <height>23</height>
+          </rect>
+        </property>
+        <property name="text">
+          <string>Focus on Voice</string>
+        </property>
+        <property name="checkable">
+          <bool>true</bool>
+        </property>
+      </widget>
+      <widget class="QCheckBox" name="ambientSoundControlCheckBox">
+        <property name="geometry">
+          <rect>
+            <x>10</x>
+            <y>30</y>
+            <width>261</width>
+            <height>23</height>
+          </rect>
+        </property>
+        <property name="text">
+          <string>Ambient Sound Control</string>
+        </property>
+        <property name="checkable">
+          <bool>true</bool>
+        </property>
+        <property name="checked">
+          <bool>true</bool>
+        </property>
+      </widget>
+    </widget>
+    <widget class="QGroupBox" name="deviceDiscoveryGroupBox">
+      <property name="geometry">
+        <rect>
+          <x>10</x>
+          <y>0</y>
+          <width>431</width>
+          <height>221</height>
+        </rect>
+      </property>
+      <property name="title">
+        <string>Device Discovery</string>
+      </property>
+      <widget class="QPushButton" name="connectButton">
+        <property name="enabled">
+          <bool>false</bool>
+        </property>
+        <property name="geometry">
+          <rect>
+            <x>10</x>
+            <y>190</y>
+            <width>101</width>
+            <height>25</height>
+          </rect>
+        </property>
+        <property name="text">
+          <string>C&amp;onnect</string>
+        </property>
+      </widget>
+      <widget class="QPushButton" name="refreshButton">
+        <property name="geometry">
+          <rect>
+            <x>120</x>
+            <y>190</y>
+            <width>131</width>
+            <height>25</height>
+          </rect>
+        </property>
+        <property name="text">
+          <string>&amp;Refresh devices</string>
+        </property>
+      </widget>
+      <widget class="QListWidget" name="deviceListWidget">
+        <property name="geometry">
+          <rect>
+            <x>10</x>
+            <y>30</y>
+            <width>411</width>
+            <height>151</height>
+          </rect>
+        </property>
+      </widget>
+    </widget>
+    <widget class="QLabel" name="label_3">
+      <property name="geometry">
+        <rect>
+          <x>10</x>
+          <y>390</y>
+          <width>131</width>
+          <height>17</height>
+        </rect>
+      </property>
+      <property name="text">
+        <string>Not endorsed by Sony!</string>
```

Comment:
This text cuts off
![image](https://user-images.githubusercontent.com/26381287/111816397-78c50c00-88e5-11eb-8299-7a96d9aa78b2.png)

Reply:
[Layout](https://doc.qt.io/qt-5/layout.html) issue. I am not an expert in the layout thing in Qt but I will try to get it working.

Reply:
Okay, awesome. I'm totally ignorant with everything regarding Qt - first time I ever read something written with it.